card/mod/05_email/set/self/never.rb

```diff
@@ -2,7 +2,7 @@
 
 self.follow_opts :position=>3
 
-self.follow_test { |opts| false }
+self.follow_test { |follower_id, accounted_ids| false }
```

Comment:
not sure whether these args are necessary to track here, but I would want to check old rubies before removing them, because some of that handling has changed.